src/labels/CustomLabels.labels

```diff
@@ -7296,6 +7472,22 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Title of checklist User Onboard in Get Started</shortDescription>
         <value>Onboard Your Users</value>
     </labels>
+    <labels>
+        <fullName>gseuChecklistWelcomeNPSPDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of Welcome to Nonprofit checklist section in Get Started</shortDescription>
+        <value>Use these essentials resources to help you get started with your NPSP journey.</value>
```

Comment:
Oops, found a typo! "Essential" should not be plural. Please change this to "Use these essential resources to help you get started with your NPSP journey."